addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -190,14 +201,14 @@ export class MassMailingHtmlField extends HtmlField {
 
     async _onSnippetsLoaded() {
         if (status(this) === 'destroyed') return;
-        if (this.wysiwyg.snippetsMenu && $(window.top.document).find('.o_mass_mailing_form_full_width')[0]) {
+        if (this.wysiwyg.state.showSnippetsMenu && $(window.top.document).find('.o_mass_mailing_form_full_width')[0]) {
```

Comment:
Turns out, this code is never executed anymore as the class `.o_mass_mailing_form_full_width` was removed by [this commit](https://github.com/odoo/odoo/commit/5226dc972560870d03e7e4bc5a838f1951b90ac7). This broke scrolling for snippets that are already in the page.
Should I make a fix for it first? I could potentially remove these lines with the fix too.

Reply:
> Should I make a fix for it first?

I guess? I did not understand what it is about. Either fix what's to be fixed or adapt so the fix can be solved later by the related team? :shrug: Whatever allows us to merge this as soon as possible :+1:

Reply:
I kept this line as is for now, but I changed the next lines to be a local state. Meaning that if the test on snippetsMenu was done so that we don't access it when it is undefined, it should be okay now to remove that part from the `if` as we can access the local state regardless.
[I asked the editor team for some clarification](https://discord.com/channels/678381219515465750/687338152372994236/1222110710595846174) before moving one though.

Reply:
> A fix in stable will still need to be done to either fully remove the dead code, or adapt it.

Removing dead code in stable should not be done. If something needs to be adapted to fix things, you can explain it to the editor team, let's focus on master :+1: